<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="loadButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="settingsButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="runButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Run simulation</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="helpButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="aboutButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>About simEC</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabMechanism">
       <attribute name="title">
        <string>Mechanism</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="mechanismLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="techniqueCB">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>50</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Voltammetry</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Chronoamperometry</string>
             </property>
            </item>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QComboBox" name="comboMechanism">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>50</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>E</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EC</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ECat</string>
             </property>
            </item>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QCheckBox" name="invertCheckbox">
            <property name="text">
             <string>Invert reaction</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="mechanismImg">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/figs/E.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabReaction">
       <attribute name="title">
        <string>Reaction</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="voltBox">
          <property name="title">
           <string>Voltammetry</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Estart</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Estartline"/>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Eswitch</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Eswitchline"/>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Scan rate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="scanrateline"/>
             </item>
             <item>
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>Shift</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="shiftEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ampBox">
          <property name="title">
           <string>Chronoamperometry</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Estart</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="EstartAmpline"/>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>tstart</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tstartline"/>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Epulse</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Epulseline"/>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>tpulse</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tpulseline"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Reaction</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Eform</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Eformline"/>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>k0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="k0line"/>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>n</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="numelline"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>alpha</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="alphaline"/>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>kcf</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="kcfline"/>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>kcr</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="kcrline"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Parameters</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Concentration</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="concline"/>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>D</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="diffcoefline"/>
             </item>
             <item>
              <widget class="QLabel" name="dblabel">
               <property name="text">
                <string>Db</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="diffcoef2line"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Area (cm2)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="arealine"/>
             </item>
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>T (K)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="templine"/>
             </item>
             <item>
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Cdl (uF/cm2)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Cdlline"/>
             </item>
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Ru (Ohm)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ruline"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabOptions">
       <attribute name="title">
        <string>Options</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QRadioButton" name="staticBt">
               <property name="text">
                <string>Static plot</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="dynamicBt">
               <property name="text">
                <string>Dynamic plot</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>First plot</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="firstplotCB">
               <item>
                <property name="text">
                 <string>i vs E</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>i vs t</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>E vs t</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>E vs i</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Second plot</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="secondplotCB">
               <item>
                <property name="text">
                 <string>None</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>i vs E</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>E vs i</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>E vs t</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>i vs t</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>Experimental data</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="expFileline"/>
               </item>
               <item>
                <widget class="QPushButton" name="browseBtn">
                 <property name="text">
                  <string>Browse</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QCheckBox" name="blankcb">
                 <property name="text">
                  <string>Overlap simulated response to blank</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_simEC">
   <property name="text">
    <string>About simEC</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
